var/spack/repos/builtin/packages/paraview/package.py

```diff
@@ -602,9 +602,9 @@ def use_x11():
         else:
             cmake_args.append("-DPARAVIEW_ENABLE_PYTHON:BOOL=OFF")
 
+        cmake_args.append("-DPARAVIEW_USE_MPI:BOOL=%s" % variant_bool("+mpi"),
```

Comment:
Might this be a good place to use `self.define_from_variant()`

Reply:
I might come back and clean up the package.py to do that for the other places, but for now I figured I could just follow the method used throughout the package.

Reply:
makes sense, consistency is preferable vs succinctness.

Reply:
@kwryankrattiger, @vicentebolea: I added a proposal to select `netcdf-c`'s +mpi/~mpi depending on `+mpi`/`~mpi` as well, which also caused `hdf5+mpi` that triggered #47097 using `spack install paraview~mpi` to use `hdf5~mpi` as well, fixing #47097 for me:
[Likewise, use ~mpi/+mpi to select netcdf-c and hdf5 mpi on/on](https://github.com/spack/spack/pull/47114/commits/19e17545adc8bb7aa297b035665687df2655986a)

Reply:
Replaced by even better conflict on `"~mpi"`: Works for the builds I tested.